environments/production/aws-resources.yml

```diff
@@ -90,6 +90,9 @@ pillow1-production.instance_id: i-0354a00e5ace39eb2
 proxy1-production: 10.202.20.47
 proxy1-production.instance_id: i-07bcff75252c1d093
 proxy1-production.public_ip: 100.24.216.221
+proxy2-production: 10.202.20.123
+proxy2-production.instance_id: i-08707523172eb37bf
+proxy2-production.public_ip: 54.173.241.40
```

Comment:
Todo: we no longer need to provision a public IP because requests are routed through global accelerator and an ALB